docs/site/Creating-components.md

```diff
@@ -202,57 +195,51 @@ When writing a custom sequence action, you need to access Elements contributed b
 
 Because all Actions are resolved before the Sequence `handle` function is run, Elements contributed by Actions are not available for injection yet. To solve this problem, use `@inject.getter` decorator to obtain a getter function instead of the actual value. This allows you to defer resolution of your dependency only until the sequence action contributing this value has already finished.
 
-```js
+```ts
 export class AuthenticationProvider {
   constructor(
     @inject.getter(BindingKeys.Authentication.STRATEGY)
     readonly getStrategy
-  ) {
-    this.getStrategy = getStrategy;
```

Comment:
I am not sure if `readonly` is enough for TypeScript to create a property called `getStrategy` and initialize it from the constructor argument (see also my comment above). Could you please double-check?

There are more places below where you are removing explicit initialization of properties from constructor arguments. I am not going to repeat my comment, please check those places yourself.

Reply:
Good catch. For TS you need to have a keyword such as `public`, `private`, etc. `readonly` alone like in this example also works just fine. In fact that's exactly how the AuthenticationProvider is implemented. See https://github.com/strongloop/loopback-next/blob/master/packages/authentication/src/providers/authenticate.ts#L51-L54